var/spack/repos/builtin/packages/catalyst/package.py

```diff
@@ -35,12 +36,12 @@ class Catalyst(CMakePackage):
     variant('extras', default=False, description='Enable Extras support')
     variant('rendering', default=False, description='Enable VTK Rendering support')
     variant('osmesa', default=True, description='Use offscreen rendering')
-    conflicts('+osmesa', when='~rendering')
+    conflicts('+osmesa', when='+rendering')
 
     depends_on('git', type='build')
     depends_on('mpi')
-    depends_on('python@2:2.8', when='+python', type=("build", "link", "run"))
-    depends_on('python', when='~python', type=("build"))
+    depends_on('python@3:', when='+python', type=("build", "link", "run"))
+    depends_on('python@3:', when='~python', type=("build"))
```

Comment:
Does catalyst actually require python as a build dep?

Reply:
Not sure, I'll check.